Formula/ammonite-repl.rb

```diff
@@ -19,7 +20,8 @@ def install
   end
 
   test do
-    output = shell_output("#{bin}/amm -c 'print(\"hello world!\")'")
-    assert_equal "hello world!", output.lines.last
+    (testpath/"testscript.sc").write("#!/usr/bin/env amm\n@main\ndef fn(): Unit = {\nprintln(\"hello world!\")\n}\n")
```

Comment:
```suggestion
    (testpath/"testscript.sc").write <<~EOS
      #!/usr/bin/env amm
      @main
      def fn(): Unit = {
      println("hello world!")
      }
    EOS
```
Why do we need to change this?

Reply:
I changed the test to one that is slightly more comprehensive. This test fails on the "bad" version of Ammonite, and passes on the "good" version.
Once the latest versions are fixed, presumably this test will pass again, and allow us to upgrade it here.